config/cars/kunos/lotus_evora_s.ini

```diff
@@ -1,6 +1,10 @@
 [INCLUDE]
 INCLUDE=common/no_popup_lights.ini
 
+[SHADER_REPLACEMENT_...]
```

Comment:
wipers are actually broken completely on this car, not sure what i meant to do here... will see if i can fix them.